proto/beacon/p2p/v1/messages.proto

```diff
@@ -11,17 +11,17 @@ import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
 enum Topic {
   UNKNOWN = 0;
-  BEACON_BLOCK_ANNOUNCE = 1;
+  BEACON_BLOCK_ANNOUNCE = 1 [deprecated=true];
   BEACON_BLOCK_REQUEST = 2;
-  BEACON_BLOCK_REQUEST_BY_SLOT_NUMBER = 3;
+  BEACON_BLOCK_REQUEST_BY_SLOT_NUMBER = 3 [deprecated=true];
   BEACON_BLOCK_RESPONSE = 4;
   BATCHED_BEACON_BLOCK_REQUEST = 5;
   BATCHED_BEACON_BLOCK_RESPONSE = 6;
```

Comment:
All of this is deprecated

Reply:
Ah ok. Wasn't sure if they were block req/resp were still valid. 

Deprecated everything